src/makeRobotAccountant.js

```diff
@@ -7,6 +7,23 @@
 
 function makeRobotAccountant() {
   // write code here
+  let sum = 0;
+  let count = 0;
+  const errorMessage = 'Bzzz... Error!';
+
+  return (x) => {
+    return (y) => {
```

Comment:
if you want, you can change x and y to firstNumber or Value and etc.

Reply:
agree, let's use more descriptive variable names